mmv1/third_party/terraform/acctest/vcr_utils.go

```diff
@@ -242,14 +264,34 @@ func initializeReleaseDiffTest(c resource.TestCase, testName string) resource.Te
 			},
 		}
 	}
+	// InsertDiffSteps adds modified steps to the test that run with an external provider
+	// these steps do the actual infrastructure provisioning, and c.Steps is updated to have these modified steps
+	c.Steps = InsertDiffSteps(c, tempOutputFile, releaseProvider, localProviderName)
```

Comment:
I'd recommend just `InsertDiffSteps(c, tempOutputFile, releaseProvider, localProviderName)`, since you're mutating the test case anyway (comment would need to be updated too)